### PR TITLE
#29986 [CUBRID] Create and edit View using UI

### LINKED ISSUE #29986: [CUBRID] Create and Edit View using UI
**Problem**
Currently, The option to create and edit View is only through script.

**Solution**
Allow user to create and edit View using UI.

### PR DESCRIPTION
**Pull request by longhaseng52**
Allow user to create and edit View using UI.

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            viewDDL = "";
+        }
+        if (!view.isPersisted()) {
+            query.append("CREATE VIEW " + view.getUniqueName() + "\nAS ");
+            query.append(viewDDL);
+            if (hasComment && view.getDescription() != null) {
+                query.append("\nCOMMENT = " + SQLUtils.quoteString(view, CommonUtils.notEmpty(view.getDescription())));
+            }
+        } else {
+            if (command.hasProperty("objectDefinitionText")) {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java]: Use `DBConstants.PARAM_OBJECT_DEFINITION_TEXT`

**Review by ShadelessFox — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java (hunk 1 above)
Comment (longhaseng52) [reply plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java]: I have fixed this.

**Comment by E1izabeth** [comment]
We passed your PR to our QA team for testing.

**Review by ShadelessFox — APPROVED** [review]

**Comment by uslss** [comment]
For now, view definition isn't available for Views after saving. Refresh and reconnect doesn't help

https://github.com/user-attachments/assets/799baff6-eb3a-41cc-a8c7-8ced4116cbaf

**Pull request merged**